terra-cluster/variables.tf

```diff
@@ -22,7 +22,7 @@ locals {
 }
 variable "service" {
   description = "App name"
-  default = "framework"
+  default = "terra"
```

Comment:
Is this optional? Can we leave this empty? Because I feel the default name as Terra sounds unnecessary..

Reply:
This variable is used for identifying the cluster, along with the TF workspace (see cluster_network local below). After some discussions on the devops team we thought Terra was a more appropriate name than Framework, but I don't really have strong feelings about this. We need to name the cluster something 🤷‍♂